package.json

```diff
@@ -4,9 +4,20 @@
   "description": "Run gherkin-syntaxed specs with cypress.io",
   "main": "index.js",
   "scripts": {
-    "test": "eslint . && jest && cypress run",
```

Comment:
this way we don't lint on CI anymore. I think it's fine to have all 3 as tests, as per:

![screen shot 2018-10-10 at 1 18 56 pm](https://user-images.githubusercontent.com/4002543/46726271-133db180-cc8f-11e8-9b71-e31c8fad0c64.png)